### ISSUE TITLE
[pose] : `delay` in transition config does not work

### ISSUE THREAD

**Issue by grundmanise** [Popmotion/popmotion#377]
**Describe the bug**
Setting `delay` in `transition` object for a pose does not add the delay.

**To Reproduce**
```
const Square = posed.div({
  hovered: {
    scale: 1.5,
    transition: {
      duration: 600,
      type: "tween",
      ease: "easeIn",
      loop: Infinity,
      delay: 1000 // <- does not change anything
    }
  }
});
```

**Expected behavior**
Set the delay before starting the transition. When loop is `Infinity`  -> delay each loop.

**Link to CodePen or similar**
https://codesandbox.io/s/xr1oo12lq4

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version: 66.0.3359.181

**Comment by InventingWithMonster** [comment]
You can set `delay` on the pose itself for now, thanks for finding this bug

**Comment by InventingWithMonster** [comment]
Fixed in `popmotion-pose` 2.0.5
